drush/site-aliases/foia.aliases.drushrc.php

```diff
@@ -22,6 +22,29 @@
   'root' => '/mnt/gfs/foia.dev/livedev/docroot',
 );
 
+if (!isset($drush_major_version)) {
+  $drush_version_components = explode('.', DRUSH_VERSION);
+  $drush_major_version = $drush_version_components[0];
+}
+
+// Site foia, environment prod.
+$aliases['prod'] = array(
+  'root' => '/var/www/html/foia.prod/docroot',
+  'ac-site' => 'foia',
+  'ac-env' => 'prod',
+  'ac-realm' => 'prod',
+  'uri' => 'foia.prod.acquia-sites.com',
+  'remote-host' => 'foia.ssh.prod.acquia-sites.com',
+  'remote-user' => 'foia.prod',
+  'path-aliases' => array(
+    '%drush-script' => 'drush' . $drush_major_version,
+  ),
+);
+$aliases['prod.livedev'] = array(
```

Comment:
Does prod have a livedev mode? I didn't think so. Did you get this alias definition from Acquia Cloud?

Reply:
@malikkotob Yes, I just copied the snippet from the downloaded version.